### ISSUE TITLE
[Unexpected]Unexpected error creating subscription

### ISSUE THREAD

**Issue by TravisSperry** [nateleavitt/infusionsoft#56]
Thoughts on how I might troubleshoot this? I'm running Ruby 2.3.1 and Rails 5.0.2.

I have access to all the other methods, data_query, data_find_by_field, invoice_create_blank_order, etc. 

```
Infusionsoft::UnexpectedError: [Unexpected]Unexpected error creating subscription
	from /Users/travis/.rvm/gems/ruby-2.3.1/gems/infusionsoft-1.1.9/lib/infusionsoft/exception_handler.rb:26:in `initialize'
	from /Users/travis/.rvm/gems/ruby-2.3.1/gems/infusionsoft-1.1.9/lib/infusionsoft/connection.rb:25:in `new'
	from /Users/travis/.rvm/gems/ruby-2.3.1/gems/infusionsoft-1.1.9/lib/infusionsoft/connection.rb:25:in `rescue in connection'
	from /Users/travis/.rvm/gems/ruby-2.3.1/gems/infusionsoft-1.1.9/lib/infusionsoft/connection.rb:16:in `connection'
	from /Users/travis/.rvm/gems/ruby-2.3.1/gems/infusionsoft-1.1.9/lib/infusionsoft/request.rb:30:in `request'
	from /Users/travis/.rvm/gems/ruby-2.3.1/gems/infusionsoft-1.1.9/lib/infusionsoft/request.rb:6:in `get'
	from /Users/travis/.rvm/gems/ruby-2.3.1/gems/infusionsoft-1.1.9/lib/infusionsoft/client/invoice.rb:75:in `invoice_add_recurring_order'
	from /Users/travis/.rvm/gems/ruby-2.3.1/gems/infusionsoft-1.1.9/lib/infusionsoft.rb:19:in `method_missing'
```

**Comment by jaredjackson** [comment]
Just saw this. These may be related. #58

**Comment by TravisSperry** [comment]
Interesting. Maybe submit a pull request? Sounds right. Do you have a working branch with the changes you made?

**Comment by nateleavitt** [comment]
This should be fixed in the latest version of the gem `1.2.0`.
